auth.go

```diff
@@ -0,0 +1,99 @@
+package winrm
+
+import (
+	"fmt"
+	"io/ioutil"
+	"strings"
+
+	"github.com/masterzen/winrm/soap"
+	"launchpad.net/gwacl/fork/http"
```

Comment:
You're still using the fork. At the minimum we should use the azure fork or better have this code run/compiled only on Go 1.7+